### PR TITLE
Feature/us15286 commentaries feed

### PR DESCRIPTION
**Pull request by bzmillerboy**
Turned the commentaries page into a feed. We need to talk about ways to give content admins ability to manage social media meta tags for pages like /commentaries or /articles.

### CODE REVIEW

File: _layouts/perspective.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 <article class="article-tpl">
 
-  <header class="commentary-header"
+  <header class="perspective-header"
   style="position: relative; background-color: rgba(0, 0, 0, 0); background-image: url('{% if page.image.url %}{{ page.image.url | imgix: site.imgix }}{% else %}{{ site.default_image | imgix: site.imgix }}{% endif %}?{{ site.imgix_params.placeholder }}');" data-optimize-bg-img data-hero-image>
```
[the remark this answers is not in the document]
Comment (bzmillerboy) [reply _layouts/perspective.html]: good catch, these have been removed

File: index.html [hunk 2]
```diff
@@ -71,14 +71,14 @@ <h2 class="collection-header text-gray-light push-bottom">most recent</h2>
...
 
-      {% if page.commentaries.docs.size != 0 %}
+      {% if page.perspectives.docs.size != 0 %}
         <div class="push-bottom soft-bottom">
-          <h2 class="collection-header push-bottom text-gray-light">fresh perspectives</h2>
-          {% assign featured_commentaries = page.commentaries.docs | slice: 0, 3 %}
-          {% for featured in featured_commentaries %}
-            {% include _media-object-list.html media_type="commentary" %}
+          <h2 class="collection-header push-bottom text-gray-light">another perspective</h2>
+          {% assign featured_perspectives = page.perspectives.docs | slice: 0, 3 %}
```
[the remark this answers is not in the document]
Comment (bzmillerboy) [reply index.html]: boom genius, done

File: perspectives.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
+---
+layout: default
+title: Perspectives
+image:
+  url: "//crossroads-media.s3.amazonaws.com/images/another-perspective-social.jpg"
+paginate:
+  perspectives:
+    per: 50
```
[the remark this answers is not in the document]
Comment (bzmillerboy) [reply perspectives.html]: I've added pagination

File: perspectives.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    sort: date desc
+excerpt: "Our thoughts on culture."
+---
+
+<div data-page="articles">
+
+  {% assign featured_perspectives = page.perspectives.docs | slice: 0, 3 %}
```
[the remark this answers is not in the document]
Comment (bzmillerboy) [reply perspectives.html]: done

File: perspectives.html [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+  <div class="container">
+    <div class="row">
+      <div class="article-tpl col-sm-8 col-sm-offset-2 push-bottom">
+        <section class="soft-top">
+          <header class="more-media-header">
+            <h3>more fresh perspectives</h3>
+          </header>
+          <div class="soft-half-top">
+            {% assign limited = site.perspectives | sort: 'published_at' | reverse | slice: 3, 20 %}
+            {% for featured in limited limit: 20 %}
```
[the remark this answers is not in the document]
Comment (bzmillerboy) [reply perspectives.html]: fixed

**Comment by bzmillerboy** [comment]
@seancdavis Feedback fixes made, ready for review. 

Note: This was great feedback and just tells me that @tcmacdonald did a terrible job reviewing my previous PR since most of this feedback passed his review. Terrible @tcmacdonald terrible 😝

**Comment by tcmacdonald** [comment]
🤔 Note to self... apply more scrutiny @bzmillerboy's PRs moving forward. 🤔

**Pull request merged**